README.md

```diff
@@ -1,70 +1,70 @@
-# Getting Started with Create React App
+# Stock Company Information
```

Comment:
- [ ] Some of your tests are failing, also please add more tests

Reply:
Hello, all tests pass and I have only three components to test plus the App.js file...